### PR TITLE
Add opacity mixin, No CSS Added!

### PR DESCRIPTION
**Pull request by IamManchanda**
I think this is a useful mixin as i don't think adding classes would be great on this infact bloat!

### CODE REVIEW

**Comment by rafibomb** [comment]
Yes to mixins!

**Comment by brettsmason** [comment]
I don't understand why we need this? Ie9 supports default opacity setting so why the need for the filter, as we don't support older browsers? It's quicker to type the style than to use the mixin :smile:

File: scss/prototype/_opacity.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+/// Opacity Mixin: Easily create opacity to an element
+/// @param {Number} $opacity[] Opacity/Transparency of an element
+@mixin opacity($opacity) {
+  @if ($opacity >= 0 and $opacity <= 1) {
+    opacity: $opacity;
+    $opacity-ie: $opacity * 100;
+    filter: alpha(opacity=$opacity-ie); //IE8
+  } @else {
+    @warn 'Opacity should be between 0.0 and 1.0';
```
Comment (IamManchanda) [inline scss/prototype/_opacity.scss]: @brettsmason But this is yelling too , if user makes a mistake!

Comment (brettsmason) [inline scss/prototype/_opacity.scss]: @IamManchanda you could say that about every single CSS property though (eg `float: top;`) 😄

**Comment by kball** [comment]
Guys I don't get this... the only value this creates is for a browser we don't support... why would we do this?

**Comment by IamManchanda** [comment]
Ohk please revert :disappointed: .... thinked about it ... yes you and @brettsmason are true!

**Pull request merged**